src/sherpa_ai/database/user_usage_tracker.py

```diff
@@ -86,8 +95,27 @@ def get_sum_of_tokens_since_last_reset(self, user_id):
         token_sum = sum(item["token"] for item in data_since_last_reset[1:])
         return token_sum
 
+    def token_left_calculator(self, combined_id):
+        split_parts = combined_id.split('_')
+        user_id = ""
+        if len(split_parts) > 0:
+            user_id = split_parts[0]
+
+        user_is_whitelisted = self.is_in_whitelist(user_id)
+        if not user_is_whitelisted and not self.is_reminded:
+            total_token_since_last_reset = self.get_sum_of_tokens_since_last_reset(
+                user_id=combined_id
+            )
+            maximum_daily_token = float(
+                getattr(cfg, 'DAILY_TOKEN_LIMIT', self.max_daily_token))
+            token_left = maximum_daily_token - total_token_since_last_reset
+            percentage_used = ((float(maximum_daily_token) - float(token_left))*100) / float(maximum_daily_token)
+            if percentage_used > 75:
```

Comment:
Right now it looks like this logging is going to happen every time we add data about token usage. We could end up with many log lines. 

Instead we should give the user a reminder just once per day.